pkg/core/localize/locales/en/cmd/context.en.toml

```diff
@@ -3,28 +3,33 @@ one='Group, share and manage your rhoas services'
 
 [context.cmd.longDescription]
 one='''
-rhoas context commands allow developers to:
+Group your application service instances into service contexts.
 
-  * Group services into contexts that can be used with a number of CLI commands.
-  * Manage different service contexts by switching, listing and removing service contexts 
-  * Share context with others to use the same set of services
-  * Generating configuration for connecting to the services from various platforms and tools
+A service context is a group of application service instances and their configuration details. By using service contexts, you can group together application service instances that you want to use together.
```

Comment:
Since contexts hold only service identifiers, I don't think we should use the term configuration details here.

Reply:
Thanks! Done.